install_pim/manual/daemon_queue.rst

```diff
@@ -30,6 +30,16 @@ You can also run the daemon to execute only one job and then exit. This is usefu
 
     $ /path/to/php /path/to/your/pim/bin/console akeneo:batch:job-queue-consumer-daemon --env=prod --run-once
 
+Another possibility is to launch several daemons that will consume different jobs. This could be useful if you want a specific job to be consume sooner.
```

Comment:
```suggestion
Another possibility is to launch several daemons that will consume different jobs. This could be useful if you want a specific job to be consumed sooner.
```